### ISSUE TITLE
CardInfo example doesn't work with nightly

### ISSUE THREAD

**Issue by Pitel** [arduino/Arduino#2868]
Works with 1.6.1, can't test with 1.6.2 because I'm on Linux (64bit).

When compiled and uploaded with 1.6.1, I get nice info about card and files. On nightly I just get this:
```
Initializing SD card...initialization failed. Things to check:
* is a card is inserted?
* Is your wiring correct?
* did you change the chipSelect pin to match your shield or module?
```

I cleaned `.arduino15` before installing nightly. Nightly's MD5 is `b20bce289c0dc848eee665f81f60b50a`.

**Comment by Pitel** [comment]
I also fixed the typos in #2869 :)

**Comment by facchinm** [comment]
Hi @Pitel ,
does it work if you add 
```arduino
digitalWrite(10, HIGH)
```
just after 
``` arduino 
pinMode(10, OUTPUT);
``` 
?

It could be related to a change (#2659) that solves a problem for someone but could create a lot of trouble to existing sketches.
I'll wait for your feedback before reverting it :smirk:

**Comment by Pitel** [comment]
Yes, it works after adding `pinMode(10, OUTPUT)`.

**Comment by vicatcu** [comment]
I've made an update regarding issue #2659 that may satisfy both concerns. See comments in the.

**Comment by Pitel** [comment]
Well, I'm quite new to Arduino, so I don't know much about this SPI thing. But my general opinion is against legacy stuff. If there is new better/faster/more correct way of doing somehting, and it's not backward compatible, then use the new way. But! Make the change as visible as possible, so that one can easily find how to adopt this new change in their old project.

**Comment by facchinm** [comment]
Fixed in #2974
